### PR TITLE
Note VS for Mac EOL

### LINKED ISSUE #30217: Add an alert box in VS for Mac sections of tutorials, to warn about impending retirement of VS for Mac
https://devblogs.microsoft.com/visualstudio/visual-studio-for-mac-retirement-announcement/

---
[Associated WorkItem - 159812](https://dev.azure.com/msft-skilling/Content/_workitems/edit/159812)

### PR DESCRIPTION
**Pull request by tdykstra**
Fixes #30217 

This is mainly about getting the notice into tutorials, which it does by adding an include to the VS Mac prerequisites includes.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/security/enforcing-ssl.md](https://github.com/dotnet/AspNetCore.Docs/blob/b81dfafa9726c785159ca0758ff3be7e341b11a7/aspnetcore/security/enforcing-ssl.md) | [Enforce HTTPS in ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/security/enforcing-ssl?branch=pr-en-us-30382) |
| [aspnetcore/test/razor-pages-tests.md](https://github.com/dotnet/AspNetCore.Docs/blob/b81dfafa9726c785159ca0758ff3be7e341b11a7/aspnetcore/test/razor-pages-tests.md) | [Razor Pages unit tests in ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/test/razor-pages-tests?branch=pr-en-us-30382) |
| [aspnetcore/tutorials/publish-to-azure-webapp-using-vs.md](https://github.com/dotnet/AspNetCore.Docs/blob/b81dfafa9726c785159ca0758ff3be7e341b11a7/aspnetcore/tutorials/publish-to-azure-webapp-using-vs.md) | [Publish an ASP.NET Core app to Azure with Visual Studio](https://review.learn.microsoft.com/en-us/aspnet/core/tutorials/publish-to-azure-webapp-using-vs?branch=pr-en-us-30382) |
| [aspnetcore/tutorials/publish-to-iis.md](https://github.com/dotnet/AspNetCore.Docs/blob/b81dfafa9726c785159ca0758ff3be7e341b11a7/aspnetcore/tutorials/publish-to-iis.md) | [[Visual Studio](#tab/visual-studio)](https://review.learn.microsoft.com/en-us/aspnet/core/tutorials/publish-to-iis?branch=pr-en-us-30382) |

### CODE REVIEW

**Review by wadepickett — APPROVED** [review]
Approved. Putting the note in the prereqs include was a good idea.  This all looks good to me here for these files and in the tutorials using the prereqs.

**Comment by wadepickett** [comment]
@tdykstra, I didn't realize we could put in meta data for the include itself.  Are we doing that for all includes going forward, or just this case?

**Comment by guardrex** [comment]
I'm 👀 on that, too, and will add it to Blazor includes if that's what we'll be doing.

btw - Is there a known release date for the C# Dev Kit? I was planning on adding it to Blazor tutorials, but I thought it best to wait given they have several open Blazor issues/problems with it.

**Review by Rick-Anderson — APPROVED** [review]

File: aspnetcore/includes/vs-mac-eol.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+author: tdykstra
+ms.author: tdykstra
+ms.date: 09/15/2023
+ms.topic: include
+---
```
Comment (Rick-Anderson) [inline aspnetcore/includes/vs-mac-eol.md:6]: Should we include this meta-data with includes? We haven't been doing that.

Comment (tdykstra) [reply aspnetcore/includes/vs-mac-eol.md:6]: Theoretically it's required, even more than I put in: https://review.learn.microsoft.com/en-us/help/platform/metadata-required-include-files?branch=main

**Comment by tdykstra** [comment]
> btw - Is there a known release date for the C# Dev Kit? I was planning on adding it to Blazor tutorials, but I thought it best to wait given they have several open Blazor issues/problems with it.

So far I've found it to be pretty buggy and in the dotnet docs tutorial I advise people to uninstall or disable it as a prerequisite for a tutorial.

File: aspnetcore/includes/vs-mac-eol.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+author: tdykstra
+ms.author: tdykstra
+ms.date: 09/15/2023
+ms.topic: include
```
Comment (Rick-Anderson) [inline aspnetcore/includes/vs-mac-eol.md:5]: Putting include files in a `/include` folder is a good convention, but I'm wondering with `ms.topic: include` if that's not a requirement.

**Pull request merged**